src/autofix/autofix.ts

```diff
@@ -264,18 +275,25 @@ export default async function ({
 		}
 	}
 
+	for (const resource of xmlResources) {
+		const resourcePath = resource.getPath();
+		const newContent = await applyFixesXml(resource, messages.get(resourcePath)!);
+		if (newContent) {
```

Comment:
Could it be that the XML syntax breaks after the fix?
I was wondering whether we could do the same thing as with JS- check somehow for syntax errors after the fix and report them

Reply:
Good point. sax-wasm might not be suitable for that purpose though. I'll check what options we have

Reply:
If you don't mind, I would like to add XML validation in a separate PR. 

Implementing it might take longer, but since it doesn't fundamentally conflict with the work done here, it is probably easier to split it, also from a code review perspective.